modules/integration/tests-integration/tests-registry/src/test/resources/testng.xml~

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
Is this a temporary file (since it is ending with ~)? if this is not required, can you remove it from the PR plz.